_templates/countdown.php

```diff
@@ -0,0 +1,9 @@
+<script src="https://code.jquery.com/jquery-3.1.0.min.js" integrity="sha256-cCueBR6CsyA4/9szpPfrX3s49M9vUU5BgtiJj06wt/s=" crossorigin="anonymous"></script>
```

Comment:
I don't understand what you're doing with the script tags here, jQuery should already be loaded by the page anyway, why are we including it again?